Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ocrd/core:v2.62.0 AS base
+FROM ocrd/core:v2.67.1 AS base
```

Comment:
```suggestion
FROM docker.io/ocrd/core:v2.67.1 AS base
```

Reply:
While updating the version, support for podman can be added, too.

Reply:
Until now we always use Dockerhub registry implicitly, so supporting Podman would require similar changes in lots of places. Also, we don't know of Dockerhub will stay free. Therefore I'd prefer a more general mechanism.

How about generating derived Dockerfiles from the existing, git-controlled Dockerfile via makefile rule? For example, we could have:

```make

Dockerfile.podman: Dockerfile
	sed "s,FROM ocrd/core,FROM docker.io/ocrd/core," < $< > $@
	#sed "s,FROM ocrd/core,FROM ghcr.io/ocr-d/core," < $< > $@

podman: Dockerfile.podman
	podman build -f $< .
```

Reply:
The suggested change would not change the status quo for Docker users, but add support for Podman here without adding complexity. ocrd_olena already uses this pattern, 13 other Dockerfiles would still need it.

Reply:
Sure, we can also do that change along the way (and come back to a more general mechanism when really needed).